interface/orders/gen_hl7_order.inc.php

```diff
@@ -37,6 +37,7 @@
 require_once("$srcdir/classes/Address.class.php");
 require_once("$srcdir/classes/InsuranceCompany.class.php");
 require_once("$webserver_root/custom/code_types.inc.php");
+require_once("../globals.php");
```

Comment:
This should not be needed since can assume it has already been included (or else this code will crap out because above $srcdir and $webserver_root directories would not be populated)